test/test_cli.rb

```diff
@@ -352,6 +352,17 @@ def test_load_path
     $LOAD_PATH.shift
   end
 
+  def test_extra_runtime_dependencies
+    failed = false
+    begin
```

Comment:
This basically amounts to `assert_nothing_raised`, which is [intentionally not part of minitest](http://www.zenspider.com/ruby/2012/01/assert_nothing_tested.html). Is there a simple positive assertion we could make here instead?

Reply:
I'll have a go at that!

Reply:
Having had a got at it, I'm grateful for the hint.